From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jonathan Leitschuh <jlleitschuh@wpi.edu>
Date: Fri, 13 Jun 2014 14:36:37 -0400
Subject: [PATCH 0099/6262] Removes the print line from the Preferences class
 so that the write method no longer causes a system print

Change-Id: I42af95c3bdd48fa1e2d92fe1c70b8e777f822616
---
 .../src/main/java/edu/wpi/first/wpilibj/Preferences.java         | 1 -
 1 file changed, 1 deletion(-)

diff --git a/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/Preferences.java b/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/Preferences.java
index fac58153b80b4237b677b95aa6e3f85b88d91ee3..558662194329b86b9a9e4beb37adb2bad03d165f 100644
--- a/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/Preferences.java
+++ b/wpilibj/wpilibJava/src/main/java/edu/wpi/first/wpilibj/Preferences.java
@@ -657,7 +657,6 @@ public class Preferences {
                                 comment = null;
                             }
 
-                            System.out.println(name + "=" + values.get(name));
 
                             if (shouldBreak) {
                                 break;
